### PR TITLE
Add reportlab recipe

### PR DESCRIPTION
**Pull request by mcs07**

### CODE REVIEW

**Comment by jakirkham** [comment]
Is there a test suite we can run? Just thinking it would be nice to do given there is some compiled code.

**Comment by mcs07** [comment]
I've added the test suite to the build scripts.

**Comment by souravsingh** [comment]
@jakirkham The recipe is ready for merging

File: recipes/reportlab/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  features:
+    - vc9   # [win and py27]
+    - vc14  # [win and py>=35]
```
Comment (ocefpaf) [inline recipes/reportlab/meta.yaml]: There are people fetching conda-forge recipes and building Python 3.4 packages.
So, if you add add the `vc10 ` here, you will make their lives easier.

However, do you need `vc`s for this package? If so please add them to `build` and `run` section below.
See http://conda-forge.github.io/docs/buildwin.html#vc-features

Comment (mcs07) [inline recipes/reportlab/meta.yaml]: @ocefpaf I believe the native extensions are only used from python, so I've removed the vc features.

**Pull request merged**